developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_28.rst

```diff
@@ -58,6 +58,7 @@ Removed APIs
 * ``OC.loadScript`` and ``OC.loadStyle``: Use ``OCP.Loader`` instead.
 * ``OC.appSettings``: There is no replacement.
 * ``OCA.Files``: Everything removed but Sidebar and Settings. See the Added API section for replacements.
+* ``OCP.Comments``: Removed comments loading.
```

Comment:
`OCA.Comments` !== `\OCP\Comments`

Reply:
Something like that, but this is the `Front-end changes` section :)
But also document the front API removal, so both sections should have a removal notice

Reply:
Ah! You're right, thanks for the diligence 🙏